sentinel-adapter/sentinel-dubbo-adapter/src/main/java/com/alibaba/csp/sentinel/adapter/dubbo/SentinelDubboProviderFilter.java

```diff
@@ -56,19 +53,20 @@ public SentinelDubboProviderFilter() {
     @Override
     public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcException {
         // Get origin caller.
-        String origin = DubboOriginParserRegistry.getDubboOriginParser().parse(invoker, invocation);
+        String origin = DubboAdapterGlobalConfig.getOriginParser().parse(invoker, invocation);
         if (null == origin) {
             origin = "";
         }
 
         Entry interfaceEntry = null;
         Entry methodEntry = null;
         try {
-            String resourceName = getResourceName(invoker, invocation, DubboConfig.getDubboProviderPrefix());
+            String methodResourceName = getMethodResourceName(invoker, invocation,
+                DubboAdapterGlobalConfig.getDubboProviderPrefix());
             String interfaceName = invoker.getInterface().getName();
-            ContextUtil.enter(resourceName, origin);
+            ContextUtil.enter(methodResourceName, origin);
             interfaceEntry = SphU.entry(interfaceName, ResourceTypeConstants.COMMON_RPC, EntryType.IN);
-            methodEntry = SphU.entry(resourceName, ResourceTypeConstants.COMMON_RPC,
+            methodEntry = SphU.entry(methodResourceName, ResourceTypeConstants.COMMON_RPC,
                 EntryType.IN, invocation.getArguments());
```

Comment:
If provider prefix is configured, the methodResourceName has a prefix which is marked for provider, it's useful when searching with prefix in dashboard menu(one scene is that only want to view all provider resources), but the interfaceResourceName has no prefix. How about also adding prefix for interface resource? 
Maybe `getInterfaceResourceName` can be added in `AbstractDubboFilter`.Same  for consumer Filter.

Besides, is there a need to distinguish between interface and method via another prefix, a little tangled. 
@sczyh30 @jasonjoo2010

Reply:
Yes, the interface resource name may be appended by the same prefix (if set).